diffkemp/semdiff/result.py

```diff
@@ -120,6 +120,46 @@ def get_symbol_names(self, compared_fun):
                     type_names.add((name, parent_name))
             return function_names, macro_names, type_names
 
+        def get_macro_defs(self, compared_fun):
+            """
+            Returns a tuple:
+            - list of macro definitions (except last macro in the callstack)
```

Comment:
I'm wondering if we should call this "macro locations" rather than "macro definitions". The latter implies to me that we're extracting macro bodies while in fact, we're just extracting locations where the macros start. OTOH, if we're using the word "definition" in this context for functions, then it's probably better to stay consistent.

Reply:
Good point, we are already using the word 'definition' eg. here https://github.com/diffkemp/diffkemp/blob/4b94c9acd38909fc5b934dabde188bd15135e05b/diffkemp/output.py#L76-L79 , so I would probably stick with the word 'definition'.